### PR TITLE
8285305: Create an automated test for JDK-4495286

### PR DESCRIPTION
**Pull request by srmandal**
Create an automated test for [JDK-4495286](https://bugs.openjdk.java.net/browse/JDK-4495286)

AccessibleJTable.setAccessibleSelction should select rows/cols when cell selection.
When cell selection is not enabled, there is no way, using
accessibility, to select rows or columns. It seems logical that selecting a cell
using accessibility should have the same effect as clicking on a cell with the
mouse. That is, if row or column selection is enabled, then selecting a cell
should instead cause the row or column to be selected.
 The proposed test verifies that the above behavior is fixed.

This review is for migrating tests from a closed test suite to open.

Testing:
The test ran successfully on Mach5 with multiple runs (40) on windows-x64, linux-x64 and macos-x64.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.java.net/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8285305](https://bugs.openjdk.java.net/browse/JDK-8285305): Create an automated test for JDK-4495286

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8333/head:pull/8333` \
`$ git checkout pull/8333`

Update a local copy of the PR: \
`$ git checkout pull/8333` \
`$ git pull https://git.openjdk.java.net/jdk pull/8333/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8333`

View PR using the GUI difftool: \
`$ git pr show -t 8333`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8333.diff">https://git.openjdk.java.net/jdk/pull/8333.diff</a>

</details>

### CODE REVIEW

**Comment by srmandal** [comment]
Hi! Can someone please take a look at this PR and provide their valuable feedback ?

File: test/jdk/javax/accessibility/4495286/AccessibleJTableSelectionTest.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2010, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+
+ * published by the Free Software Foundation.
```
Comment (mrserb) [inline test/jdk/javax/accessibility/4495286/AccessibleJTableSelectionTest.java:8]: typo

Comment (srmandal) [reply test/jdk/javax/accessibility/4495286/AccessibleJTableSelectionTest.java:7]: Fixed.

**Review by prrace — CHANGES_REQUESTED** [review]
Get rid of the useless extra directory level

**Comment by srmandal** [comment]
> Get rid of the useless extra directory level

Done.

**Comment by srmandal** [comment]
Hi! Can someone please take a look at this PR and provide their valuable feedback & approval?

**Pull request closed without merging**